<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOFD">
    <property name="title">
     <string>OFD文件(&amp;F)</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
    <addaction name="action_close"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_cut"/>
    <addaction name="action_find"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>文本(T)</string>
     </property>
     <addaction name="action_bold"/>
     <addaction name="action_italics"/>
     <addaction name="action_underline"/>
     <addaction name="action_strickout"/>
     <addaction name="action_online"/>
     <addaction name="separator"/>
     <addaction name="action_superscript"/>
     <addaction name="action_subscript"/>
    </widget>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>对齐(N)</string>
     </property>
     <addaction name="action_justifyLeft"/>
     <addaction name="action_justifyCenter"/>
     <addaction name="action_justifyRight"/>
     <addaction name="action_justifyFull"/>
    </widget>
    <widget class="QMenu" name="menu_S">
     <property name="title">
      <string>间距(S)</string>
     </property>
     <addaction name="action1_0_C"/>
     <addaction name="action1_5_C"/>
     <addaction name="action2_C"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="menu_2"/>
    <addaction name="menu_S"/>
   </widget>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>插入(&amp;I)</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>表格</string>
     </property>
     <addaction name="action_drawTable"/>
    </widget>
    <addaction name="action_images"/>
    <addaction name="menu_6"/>
    <addaction name="action_pageNumber_2"/>
   </widget>
   <widget class="QMenu" name="menu_L">
    <property name="title">
     <string>页面布局(&amp;L)</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>纸张大小</string>
     </property>
     <addaction name="action_A4"/>
     <addaction name="action_A3"/>
     <addaction name="action_A2"/>
     <addaction name="action_A1"/>
     <addaction name="action_A5"/>
     <addaction name="action_A6"/>
    </widget>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_aboutOFD"/>
   </widget>
   <addaction name="menuOFD"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_F"/>
   <addaction name="menu_L"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_saveAs"/>
   <addaction name="separator"/>
   <addaction name="action_copy"/>
   <addaction name="action_paste"/>
   <addaction name="action_cut"/>
   <addaction name="action_find"/>
   <addaction name="separator"/>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
   <addaction name="separator"/>
   <addaction name="action_images"/>
   <addaction name="action_color"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="action_new">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/new.svg</normaloff>:/images/new.svg</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建(N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/open.svg</normaloff>:/images/open.svg</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/document-save-as.svg</normaloff>:/images/document-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/document-save-all.svg</normaloff>:/images/document-save-all.svg</iconset>
   </property>
   <property name="text">
    <string>另存(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存(SaveAs)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit-copy.svg</normaloff>:/images/edit-copy.svg</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>复制(C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit-paste.svg</normaloff>:/images/edit-paste.svg</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>粘贴(P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit-cut.svg</normaloff>:/images/edit-cut.svg</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>剪切(T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_find">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit-find.svg</normaloff>:/images/edit-find.svg</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>查找(F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit-undo.svg</normaloff>:/images/edit-undo.svg</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
   <property name="toolTip">
    <string>撤销(Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit-redo.svg</normaloff>:/images/edit-redo.svg</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;Y)</string>
   </property>
   <property name="toolTip">
    <string>恢复(Y)</string>
   </property>
  </action>
  <action name="action_images">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/insert-image.svg</normaloff>:/images/insert-image.svg</iconset>
   </property>
   <property name="text">
    <string>图像(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>图像(I)</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/close.svg</normaloff>:/images/close.svg</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>关闭(C)</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/exit.svg</normaloff>:/images/exit.svg</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
   <property name="toolTip">
    <string>退出(E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/format-text-bold.svg</normaloff>:/images/format-text-bold.svg</iconset>
   </property>
   <property name="text">
    <string>粗体(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>粗体(A)</string>
   </property>
  </action>
  <action name="action_italics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/format-text-italic.svg</normaloff>:/images/format-text-italic.svg</iconset>
   </property>
   <property name="text">
    <string>斜体(&amp;B)</string>
   </property>
   <property name="toolTip">
    <string>斜体(B)</string>
   </property>
  </action>
  <action name="action_underline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/format-text-underline.svg</normaloff>:/images/format-text-underline.svg</iconset>
   </property>
   <property name="text">
    <string>下划线(&amp;D)</string>
   </property>
   <property name="toolTip">
    <string>下划线(D)</string>
   </property>
  </action>
  <action name="action_strickout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/format-text-strikethrough.svg</normaloff>:/images/format-text-strikethrough.svg</iconset>
   </property>
   <property name="text">
    <string>删除线(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>删除线(F)</string>
   </property>
  </action>
  <action name="action_online">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/under_line.png</normaloff>:/images/under_line.png</iconset>
   </property>
   <property name="text">
    <string>上划线(&amp;G)</string>
   </property>
   <property name="toolTip">
    <string>上划线(G)</string>
   </property>
  </action>
  <action name="action_superscript">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/format-text-superscript.svg</normaloff>:/images/format-text-superscript.svg</iconset>
   </property>
   <property name="text">
    <string>上标 </string>
   </property>
   <property name="toolTip">
    <string>上标</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Ctrl+P</string>
   </property>
  </action>
  <action name="action_subscript">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/text_subscript.svg</normaloff>:/images/text_subscript.svg</iconset>
   </property>
   <property name="text">
    <string>下标</string>
   </property>
   <property name="toolTip">
    <string>下标</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="action_justifyLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/format-justify-left.svg</normaloff>:/images/format-justify-left.svg</iconset>
   </property>
   <property name="text">
    <string>左对齐(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>左对齐(A)</string>
   </property>
  </action>
  <action name="action_justifyCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/format-justify-center.svg</normaloff>:/images/format-justify-center.svg</iconset>
   </property>
   <property name="text">
    <string>居中(&amp;B)</string>
   </property>
   <property name="toolTip">
    <string>居中(B)</string>
   </property>
  </action>
  <action name="action_justifyRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/format-justify-right.svg</normaloff>:/images/format-justify-right.svg</iconset>
   </property>
   <property name="text">
    <string>右对齐(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>右对齐(C)</string>
   </property>
  </action>
  <action name="action_justifyFull">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/format-justify-fill.svg</normaloff>:/images/format-justify-fill.svg</iconset>
   </property>
   <property name="text">
    <string>两端对齐(&amp;D)</string>
   </property>
   <property name="toolTip">
    <string>两端对齐(D)</string>
   </property>
  </action>
  <action name="action_A4">
   <property name="text">
    <string>&amp;A4   210毫米×297毫米</string>
   </property>
   <property name="toolTip">
    <string>A4   209毫米×296毫米</string>
   </property>
  </action>
  <action name="action_A3">
   <property name="text">
    <string>A3   &amp;297毫米×420毫米</string>
   </property>
   <property name="toolTip">
    <string>A3   296毫米×419.1毫米</string>
   </property>
  </action>
  <action name="action_A2">
   <property name="text">
    <string>A2   &amp;420毫米×594毫米</string>
   </property>
   <property name="toolTip">
    <string>A2</string>
   </property>
  </action>
  <action name="action_A1">
   <property name="text">
    <string>A&amp;1   &amp;594毫米×841毫米</string>
   </property>
   <property name="toolTip">
    <string>A1</string>
   </property>
  </action>
  <action name="action_A5">
   <property name="text">
    <string>A&amp;5   &amp;148毫米×210毫米</string>
   </property>
   <property name="toolTip">
    <string>A5</string>
   </property>
  </action>
  <action name="action_A6">
   <property name="text">
    <string>A&amp;6   &amp;105毫米×148毫米</string>
   </property>
   <property name="toolTip">
    <string>A6</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;About(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>About(A)</string>
   </property>
  </action>
  <action name="action_aboutOFD">
   <property name="text">
    <string>About &amp;OFD</string>
   </property>
   <property name="toolTip">
    <string>About OFD</string>
   </property>
  </action>
  <action name="action_color">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/color-management.svg</normaloff>:/images/color-management.svg</iconset>
   </property>
   <property name="text">
    <string>颜色(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>颜色(C)</string>
   </property>
  </action>
  <action name="action_drawTable">
   <property name="text">
    <string>表格绘制</string>
   </property>
   <property name="toolTip">
    <string>表格绘制</string>
   </property>
  </action>
  <action name="action1_5_C">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/text_line_spacing.svg</normaloff>:/images/text_line_spacing.svg</iconset>
   </property>
   <property name="text">
    <string>1.5倍行距</string>
   </property>
   <property name="toolTip">
    <string>1.5倍行距</string>
   </property>
  </action>
  <action name="action1_0_C">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/text_line_spacing.svg</normaloff>:/images/text_line_spacing.svg</iconset>
   </property>
   <property name="text">
    <string>1.0倍行距</string>
   </property>
   <property name="toolTip">
    <string>1.0倍行距</string>
   </property>
  </action>
  <action name="action2_C">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/text_line_spacing.svg</normaloff>:/images/text_line_spacing.svg</iconset>
   </property>
   <property name="text">
    <string>2.0倍行距</string>
   </property>
   <property name="toolTip">
    <string>2.0倍行距</string>
   </property>
  </action>
  <action name="action_lineSpacing">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/text_line_spacing.svg</normaloff>:/images/text_line_spacing.svg</iconset>
   </property>
   <property name="text">
    <string>行距:   1(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>行距:1(A)</string>
   </property>
  </action>
  <action name="action_pageNumber_2">
   <property name="text">
    <string>显示页码</string>
   </property>
   <property name="toolTip">
    <string>显示页码</string>
   </property>
  </action>
  <action name="action_insertTable_2">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/insert-table.svg</normaloff>:/images/insert-table.svg</iconset>
   </property>
   <property name="text">
    <string>插入表格</string>
   </property>
   <property name="toolTip">
    <string>插入表格(I)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
